src/utils/contracts.js

```diff
@@ -0,0 +1,25 @@
+import crypto from 'crypto'
+
+export function extractBodyCode(rawBytecode) {
+  return splitCode(rawBytecode).body
+}
+
+export function extractConstructorCode(rawBytecode) {
+  return splitCode(rawBytecode).constructor
+}
+
+export function splitCode(rawBytecode) {
+  const bytecode = rawBytecode.replace(/^0x/, '')
+  const constructorEndingOpcodes = '6000396000f300'
+  const endingConstructorIndex = bytecode.indexOf(constructorEndingOpcodes) + constructorEndingOpcodes.length
```

Comment:
But it would like attacking your own project... However, this function is being used with your local build files, we are not using it against the deployed contracts